.github/workflows/ci.yaml

```diff
@@ -2,24 +2,35 @@ name: GitHub Actions CI
 
 on:
   push:
-    branches: [main]
-  pull_request: {}
+    branches:
+      - main
+      - release-v*
+  pull_request:
+    branches:
+      - main
+      - release-v*
```

Comment:
docs: Not sure if you've done this in your other PR but will need to update our [release.md](https://github.com/integrations/terraform-provider-github/blob/main/RELEASE.md) to reflect that the branches need to be named `release_v#.#.#`

Reply:
I've updated the _RELEASE.md_ documentation and the pre-merge actions in the PR description to make it work.